src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,32 @@
+import classNames from 'classnames';
+import { useState } from 'react';
+import { ErrorTypes } from '../../types/ErrorTypes';
+
+type Props = {
+  error: ErrorTypes | null,
+};
+
+export const Notification: React.FC<Props> = ({ error }) => {
+  const [isNotificationVisible, setIsNotificationVisible] = useState(true);
+
+  const hideNotification = () => {
+    setIsNotificationVisible(false);
+  };
+
+  return (
+    <div className={classNames(
+      'notification is-danger is-light has-text-weight-normal',
+      { hidden: isNotificationVisible === false },
+    )}
+    >
+      <button
+        type="button"
+        className="delete"
+        aria-label="Mute volume"
```

Comment:
Not fixed 
![image](https://user-images.githubusercontent.com/70009205/234919846-a9a53b0e-2cf5-466f-8f0e-fadb3571b930.png)